recipes/perspective/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "perspective" %}
+{% set pypi_name = "perspective-python" %}
+{% set version = "0.4.2.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: debc879c1e1fbb82f57fc55beb924083f1576e7d2ddcf4d0683564ba01546a12
+
+build:
+  number: 0
+  skip: True  # [py<37 or (win and vc<14)]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - pybind11
+    - boost
+    - numpy >=1.13.1
+    - pyarrow ==0.15.1
+  run:
+    - python
+    - ipywidgets >=7.5.1
+    - {{ pin_compatible('numpy') }}
+    - python-dateutil >=2.8.0
+    - six >=1.11.0
+    - traitlets >=4.3.2
+
+test:
+  imports:
+    - perspective
```

```suggestion
    - perspective
    - perspective.table
```

@timkpaine what about adding an import to catch missing or bad c++ bindings? Not sure which is the most appropriate import (in terms of coverage and being stable).